scalableminds/webknossos
# Rescale layers for different voxel sizes on remote import

### URL of deployed dev instance (used for testing):
- https://differentvoxelsizes.webknossos.xyz

### Steps to test:
- See #6821 (add color, then add clahe as another layer -> notice that they are aligned in the resulting dataset)

### To-dos:
- [x] Frontend @philippotto 

### Issues:
- fixes #6821 

------
(Please delete unneeded items, merge only when none are left open)
